### PR TITLE
notification to user about the tokens that are being searched

### PR DESCRIPTION
**Pull request by prudhvije**
Example Format of Notifcation:
Querying http://labs.imaginea.com/betterdocs for: [java.util.regex.Matcher, java.util.regex.Pattern, java.io.FileReader, java.io.File, java.io.FileNotFoundException]

### CODE REVIEW

**Comment by prudhvije** [comment]
This will close #87

File: plugins/idea/betterdocsidea/src/main/java/com/imaginea/betterdocs/RefreshAction.java [hunk 1]
```diff
@@ -47,6 +50,9 @@
     protected static final int SIZE_DEFAULT_VALUE = 30;
     private static final String EDITOR_ERROR = "Could not get editor any active editor";
```
Comment (ScrapCodes) [inline plugins/idea/betterdocsidea/src/main/java/com/imaginea/betterdocs/RefreshAction.java]: PLease change the text to => "Could not get any active editor";

**Comment by ScrapCodes** [comment]
@amit-shar Please review this pull request.

**Comment by prudhvije** [comment]
Edited Error Message.

**Comment by ScrapCodes** [comment]
merged

**Pull request closed without merging**